daboizzBot/bot.py

```diff
@@ -0,0 +1,94 @@
+
+import discord
+import json
+from discord.ext import commands
+from discord.utils import get
+from discord_slash import SlashCommand
+
+
+def create_bot():
+    #Define prefix for given server
+    def get_prefix(client, message):
+        with open('prefixes.json', 'r') as f:
+            prefixes = json.load(f)
+        return prefixes[str(message.guild.id)]
+
+    bot = commands.Bot(command_prefix=get_prefix)
+    slash = SlashCommand(bot, sync_commands=True)
+    guild_ids = [713646548436910116]
+    bot.remove_command("help")
+
+    #When Bot Is Ready
+    @bot.event
+    async def on_ready():
+        await bot.change_presence(status=discord.Status.online, activity=discord.Game('&ping To @da boiz'))
+        print('Bot is ready')
+
+    #Prefixes
+    @bot.event
+    async def on_guild_join(guild):
+        with open('prefixes.json', 'r') as f:
+            prefixes = json.load(f)
+        prefixes[str(guild.id)] = '&'
+        with open('prefixes.json', 'w') as f:
+            json.dump(prefixes, f, indent=4)
```

Comment:
This logic is repeated with slight changes on certain lines. This should be commonised into a single function. 

```Python
def change_prefixes(function, **kwargs):
    with open('prefixes.json', 'r') as f:
        prefixes = json.load(f)
    function(prefixes, **kwargs)
    with open('prefixes.json', 'w') as f:
       json.dump(prefixes, f, indent=4)
```

Something like the above. 


Also the constant writing to and from a file seems really bad practice. Why not load the json in once to a dict and use the dict for the rest of time?